packages/commonwealth/client/scripts/state/api/threads/fetchThreads.ts

```diff
@@ -35,6 +35,7 @@ interface FetchBulkThreadsProps extends CommonProps {
   topicId?: number;
   stage?: string;
   includePinnedThreads?: boolean;
+  onArchivePage?: boolean;
```

Comment:
I suggest renaming it to something bool'ish => `is{_something_}` , `should{__something}` =>  `isArchived` etc

NAB: the `includePinnedThreads` could be renamed, I think I added that originally 😅

Reply:
Done!